### PR TITLE
List members and chair of bootstrap technical committee

### PR DESCRIPTION
**Pull request by fdegir**
### Applicable Issues

#63 

### Description of the Change

This commit updates governance document, listing the members of Bootstrap Technical Committee (BTC) and chair who have been appointed during the Technical Committee (TC) meeting on [September 14th](https://hackmd.io/SCImga0nS1qSh3QvsEOAVQ?view#September-14-2020).

### Alternate Designs

N/A

### Benefits

Increased visibility and transparency.

### Possible Drawbacks

N/A

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Fatih Degirmenci fatih.degirmenci@est.tech

### CODE REVIEW

**Comment by chr-madsen** [comment]
Accepted. Do I need to do anything now for the request to be merged?

**Review by k-hallen-ericsson — APPROVED** [review]

**Comment by fdegir** [comment]
@chr-madsen Thanks Christian!

> Do I need to do anything now for the request to be merged?

Since this repo hosts community documentation such as governance, project lifecycle and so on, most of the PRs/issues require approval from simple majority of TC members. (7 out of 12 based on the current TC composition) So we wait for review/approval of other TC members and then we merge the PR.

Ps. Some PRs such as repo setup (e.g. gitowners) can be accepted with single approval. We may need to document this somehow.

**Review by e-backmark-ericsson — APPROVED** [review]
LGTM

**Comment by e-backmark-ericsson** [comment]
One comment that came to my mind just now was that before we merge this PR we should make sure that all people mentioned in it are aware of them being mentioned there. For example I miss any comment from Ewelina, and as she was not present on the BTC forming meeting this Monday I think we should not have her mentioned there unless she approves it.

**Comment by k-hallen-ericsson** [comment]
@e-backmark-ericsson I have talked to Ewelina and told her that she was elected. ok @ewelinawilkosz ?

**Review by fredjn — APPROVED** [review]
looks good.

**Review by t031276 — APPROVED** [review]
Looking good

**Review by sodero — APPROVED** [review]

**Comment by fdegir** [comment]
@eiffel-community/eiffel-community-maintainers 

Even though the governance states simple majority is sufficient for changes like this to pass, it is important this PR to be approved by all appointed TC members. 

The reason for this is that  all the individuals will be added as TC members and this fact will publicly be available. We need to record the fact that the individuals listed in the document actually acknowledged/accepted to take part in the TC since some of them didn't join the meeting where they were nominated and didn't have chance to provide their opinions.

**Review by m-linner-ericsson — APPROVED** [review]

**Comment by ewelinawilkosz** [comment]
Sorry for being silent - busy with other engagements, working on fixing that
I'd love to a part of the BTC but I feel a bit like a joke: yes, I want to learn more about Eiffel and contribute and help improving it in the future but I am nowhere close, if nothing else my technical knowledge of the tool is still too poor and I want to make sure you're aware of that

If BTC can benefit from having someone with previous bigger open source project experience then I can definitely help but if you prefer someone more advanced technically feel free to cross me of the list

**Review by henning-roos — APPROVED** [review]

**Comment by k-hallen-ericsson** [comment]
@ewelinawilkosz please stay with us, at this stage their will be more open source-setup work than detailed technical decisions.

**Comment by fdegir** [comment]
I see we have acks/acceptances of 10 TC members and lack votes from @d-stahl-ericsson and @raja-maragani. 

Daniel and Raja were [in the meeting](https://hackmd.io/SCImga0nS1qSh3QvsEOAVQ?view#Participants1) where they were nominated and appointed as members to the TC so we can perhaps keep the PR until tomorrow to give them an extra day so they can review & comment.

**Review by raja-maragani — APPROVED** [review]

**Comment by fdegir** [comment]
Thank you all for reviews.

It is great to be with you in this journey and welcome again!

**Pull request merged**
